app/src/main/java/org/dhis2/App.java

```diff
@@ -117,7 +116,9 @@ public void onCreate() {
         MapController.Companion.init(this);
 
         setUpAppComponent();
-        Timber.plant(BuildConfig.DEBUG ? new DebugTree() : new ReleaseTree(appComponent.injectCrashReportController()));
+        if(BuildConfig.DEBUG){
```

Comment:
There are changes in the PR that are not related to the issue, remember rebase with develop